control-plane/connect-inject/constants/constants.go

```diff
@@ -2,7 +2,8 @@ package constants
 
 const (
 	// ConsulCAFile is the location of the Consul CA file inside the injected pod.
-	ConsulCAFile = "/consul/connect-inject/consul-ca.pem"
+	ConsulCAFile        = "/consul/connect-inject/consul-ca.pem"
+	ConsulCAFileWindows = "C:\\consul\\connect-inject\\consul-ca.pem"
```

Comment:
Is it guaranteed that the path will be in `C` drive ? 🤔

Reply:
connect inject directory is set up here - https://github.com/hashicorp/consul-k8s/blob/c2403adbf03e7160506143fcbc6b872c3d644713/control-plane/connect-inject/webhook/container_init.go#L45-L48

Reply:
What if the default drive is `D` and there is no `C` drive?
Maybe this is a silly question as I do not have much experience with consul on windows 😅

Reply:
generally it is C:\\ where kubernetes also mounts service token.